pkg/config/config.go

```diff
@@ -1027,6 +1032,8 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 		}
 		c.HooksDir = hooksDirs
 
+		cdi.GetRegistry(cdi.WithSpecDirs(c.CDISpecDirs...))
```

Comment:
Question: Does this ensure that subsequent calls to `cdi.GetRegistry` (as in the previous commit) use the same spec directories. If not, how does this setting affect which specs are used for device resolution.

Reply:
Yes, it should. The registry is a singleton and it's getter will reconfigure it(self) whenever if it is passed options which are different than the current set. If the options are omitted, it goes on with the current ones.